src/app.js

```diff
@@ -28,6 +28,27 @@ app.get('/', function (req, res) {
     routes.root(req, res, db);
 });
 
+app.get('/global-stats.html', function (req, res) {
```

Comment:
I don't see why we need to keep `.html` in the url - feels a bit artificial since it's not a html document? :)